libres/lib/enkf/local_obsdata_node.cpp

```diff
@@ -104,10 +103,7 @@ local_obsdata_node_get_copy_active_list(const local_obsdata_node_type *node) {
 
 bool local_obsdata_node_tstep_active(const local_obsdata_node_type *node,
                                      int tstep) {
-    if (node->all_timestep_active)
-        return true;
-    else
-        return local_obsdata_node_has_tstep(node, tstep);
```

Comment:
This behavior is going to be a bit confusing no matter what we do, I propose changing it like this, meaning that all will be active, but all individual will be off. The alternative behavior that this replaces I would say is more confusing, that everything is active, despite not existing.

Reply:
This means that this will pass on `main`:
```python
def test_tstep():
    node = LocalObsdataNode("KEY")
    assert node.allTimeStepActive()
    assert node.tstepActive(10)
    assert node.tstepActive(0)
    assert node.tstepActive(456)

    node.addTimeStep(10)
    assert not node.allTimeStepActive()
    assert node.tstepActive(10)
    assert not node.tstepActive(0)
```